.cargo/config

```diff
@@ -1,7 +1,6 @@
 [alias]
 wasm = "build --release --lib --target wasm32-unknown-unknown"
 wasm-debug = "build --lib --target wasm32-unknown-unknown"
-t = "test --workspace --exclude integration_tests"
-unit-test = "test --workspace --exclude integration_tests"
+t = "test --workspace"
+unit-test = "test --workspace"
 integration-test = "test --package integration-tests -- --ignored --test-threads 1"
-integration-test-slow = "test --package integration-tests --features slow-tests -- --ignored --test-threads 1"
```

Comment:
I decided to just remove the slow integration tests step since we arent using it anymore.